torchao/quantization/quant_api.py

```diff
@@ -117,13 +117,50 @@ def apply_dynamic_quant(model, filter_fn=None):
     change_linear_weights_to_int8_dqtensors(model, filter_fn)
 
 
-def _get_subclass_inserter(cls, **kwargs):
+import torch.nn.utils.parametrize as parametrize
+
+
+class ConstructTensorSubclass(torch.nn.Module):
+    def __init__(self, tensor_subclass_ctr, transposed, shape, groupsize, inner_k_tiles, dtype):
+        super().__init__()
+        self.tensor_subclass_ctr = tensor_subclass_ctr
+        self.transposed = transposed
+        self.shape = shape
+        self.groupsize = groupsize
+        self.inner_k_tiles = inner_k_tiles
+        self.dtype = dtype
```

Comment:
I think the usual way would be to get all of these from the tensor_subclass_instance in the right_inverse.

Reply:
you mean assign these args `self` in `right_inverse`?

Reply:
Yes